addons/mass_mailing/static/src/js/tours/mass_mailing_editor_tour.js

```diff
@@ -10,16 +10,41 @@ odoo.define('mass_mailing.mass_mailing_editor_tour', function (require) {
         trigger: '.o_app[data-menu-xmlid="mass_mailing.mass_mailing_menu_root"]',
     }, {
         trigger: 'button.o_list_button_add',
-    }, {
-        trigger: 'input[name="subject"]',
-        run: 'text Test',
```

Comment:
why this change ?

Reply:
This step is moved at the end of the tour, because I want to do a "Save" with an empty required field (the subject), so that the fixed flow can be tested